### ISSUE TITLE
Some strings are not available in translation files

### ISSUE THREAD

**Issue by FirasH** [synfig/synfig#2267]
**Synfig version & platform**:
Current translation files on Transifex platform

**Issue description**:
I've noticed that some strings are not available for translation, opening this issue to keep track of this.
Not sure if that's because the translation files currently loaded on Transifex are out of date or some fixes are required.

For example I cannot find the strings for the `Skeleton Tool` like "Bone Width:" or "Skeleton Creation".

**Comment by morevnaproject** [comment]
Hi! Those strings are not yet part of stable release, so they are not uploaded to Transifex. 
We upload strings only for stable branch (the current candidate is 1.6.0 here - https://github.com/synfig/synfig/pull/2263).

**Comment by FirasH** [comment]
@morevnaproject I’m always using master, somehow was thinking that 1.6.0 would be a snapshot of master branch 👍

Closing as resolved!

**Comment by rodolforg** [comment]
> somehow was thinking that 1.6.0 would be a snapshot of master branch

I thought it too. I still don't get it why it is not like that :)
That' how I imagined it was:

![synfig-repository-branch-design](https://user-images.githubusercontent.com/1323987/129457499-62f68bfc-a1f2-409b-983a-98ddd35cf52f.png)

**Comment by morevnaproject** [comment]
> Or something like this: (at some point, we stop to place new features to dev releases: and this is the time we branch to a new stable release-candidate branch. If we accept PR for new features, they go to master branch only.
> 
> ![synfig-repository-branch-design-2](https://user-images.githubusercontent.com/1323987/129459575-d5c95155-e797-4bf0-974b-a8c035b85c91.png)

Yes, this is exactly how it works. ^__^
At the point of 1.3.12 we decided that it is time to make a stable release, so we separated branch (1.4.x) at this point and marked all future commits as belonging to next dev cycle (1.5.0).

But after release of 1.3.12 there were some bugs found. The bugs were fixed and backported to 1.4.x. After that we made one more dev release to allow users test the fixes - 1.3.13. In this version more bugs were discovered, so we continued polishing-and-releasing process until we reached 1.4.0. ^__^



> > somehow was thinking that 1.6.0 would be a snapshot of master branch
> 
> I thought it too. I still don't get it why it is not like that :)

Well, let me explain my current thoughts. ^__^

Initially, I [wanted](https://github.com/synfig/synfig/issues/2247#issuecomment-896394838) to separate 1.6.0 branch at some point in the middle between 1.3.2 and current master branch. For the following reasons:
- I was afraid that there are too many changes introduced and wanted to deliver them to users in 2 or 3 phases (with different stable releases).
- Also I was afraid that writing Release Notes with this big amount of features will take a long time.

So, this is why I have separated 1.6.0 branch from commit 27c7b95.
And made this PR - https://github.com/synfig/synfig/pull/2263

**BUT**, then I prepared development release 1.5.0 and... and I realized that the gap between my current proposal for 1.6.0 and current dev release 1.5.0 is not that big. 
I mean, the version 1.6.0 has almost all major features, with exception of Bone Tool and maybe 1-2 other features (also master branch has many other good fixes).

So, after realizing this, I admit that my decision to separate 1.6.0 branch from "some middle point" was not a good strategy (especially considering the time wasted on cherry-picking individual commits and resolving conflicts).
It is really better to make 1.6.0 release from the top of master branch.

Proceeding with this strategy now. ^__^
